src/hint_deploy.py

```diff
@@ -0,0 +1,123 @@
+import docker
+import getpass
+import math
+import re
+import requests
+import time
+
+import constellation
+import constellation.config as config
+import constellation.docker_util as docker_util
+
+
+class HintConfig:
+    def __init__(self, path):
+        dat = config.read_yaml("{}/hint.yml".format(path))
+        self.network = config.config_string(dat, ["docker", "network"])
+        self.prefix = config.config_string(dat, ["docker", "prefix"])
+        default_tag = config.config_string(dat, ["docker", "default_tag"],
+                                           True, "master")
+        self.redis_tag = config.config_string(dat, ["redis", "tag"],
+                                              True, default_tag)
+        self.db_tag = config.config_string(dat, ["db", "tag"],
+                                           True, default_tag)
+        self.hint_tag = config.config_string(dat, ["hint", "tag"],
+                                             True, default_tag)
+        self.hint_expose = config.config_boolean(dat, ["hint", "expose"],
```

Comment:
This is like orderly-web-deploy's `test_mode` for directly exposing `hint` to the network rather than going through the proxy. Once the proxy is included we would not want to expose it.  That said, I see it's not yet used :-/

Reply:
ok, see e94cd3e